### ISSUE TITLE
how to use baidu wrap-ctc on tensorflow?

### ISSUE THREAD

**Issue by guoying1030** [tensorflow/tensorflow#7489]
when i use tensorflow ctc, i find it is very slow,,,
so i want use baidu wrap-ctc on tensorflow,

but how to write code?

when i use tensorflow ctc, i write it:
    print logits.get_shape()
    logits = tf.reshape(logits, [batch_size, -1, nout])
    logits = tf.transpose(logits, (1, 0, 2))
    print 'logits shape is '
    print logits.get_shape()
    labels = tf.SparseTensor(indices=label_idx, values=label_vals, shape=label_shape)
    print 'init label'
    loss = ctc_ops.ctc_loss(inputs=logits, labels=labels, sequence_length=seq_len)
    print 'init loss'
    cost = tf.reduce_mean(loss)
    print 'init cost'
    decoded, log_prob = ctc_ops.ctc_greedy_decoder(logits, seq_len)
    print 'init decoded'
    ler = tf.reduce_mean(tf.edit_distance(tf.cast(decoded[0], tf.int32),
                                          labels))
    print 'init ler'

**Comment by yaroslavvb** [comment]
can you file this issue with maintainers of baidu ctc?
